download.py

```diff
@@ -7,51 +7,51 @@
 #updateCoverPictures()
 def updateCoverPictures():
     cur = conn.cursor()
-    class MyOpener(urllib.FancyURLopener):
-        version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
-    myopener = MyOpener()
+    # class MyOpener(urllib.FancyURLopener):
+    #     version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
+    # myopener = MyOpener()
     cur.execute('SELECT movie, url, cover from Movies')
     for row in cur:
         movie = row[0]
         url = row[1]
         if row[2] == None:
-            print "Working on ", movie
-            content = myopener.open(url).read()
+            print("Working on ", movie)
+            content = requests.get(url)
             try:
                 cover = re.findall('<img data-retina="" src="(.+?)"',content)[0]
                 cur1 = conn.cursor()
                 cur1.execute('UPDATE Movies SET cover=? WHERE movie=?',(cover,movie))
                 conn.commit()
             except:
-                print "Error Fetching Cover photo of ",movie
+                print("Error Fetching Cover photo of ",movie)
 
 # getAllSongsList()
 def getAllSongsList():
     cur = conn.cursor()
-    class MyOpener(urllib.FancyURLopener):
-        version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
-    myopener = MyOpener()
+    # class MyOpener(urllib.FancyURLopener):
+    #     version = 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'
+    # myopener = MyOpener()
     try:
         cur.execute('select * from Songs')
     except:
-        print "Creating Songs Table"
+        print("Creating Songs Table")
         cur.execute('CREATE TABLE Songs (song_id INTEGER PRIMARY KEY AUTOINCREMENT, movie_id INTEGER, song TEXT, url TEXT, UNIQUE(movie_id,song), FOREIGN KEY (movie_id) REFERENCES Movies(movie_id))')
     cur.execute('SELECT * FROM Movies')
     for row in cur:
         movie = row[0]
         movieNm = row[1]
         url = row[2]
-        print "Working on ",movieNm,"..."
+        print("Working on ",movieNm,"...")
         cur1 = conn.cursor()
         cur1.execute('SELECT song FROM Songs WHERE movie_id = ?',(movie,))
         if not cur1.fetchone():
-            content = myopener.open(url).read()
+            content = requests.get(url)
             links = re.findall('<td class="songs-bitrate-1"><i class="fa fa-music"></i>(.+?)<em>',content)
```

Comment:
`content.text`

Reply:
Please Find the following changes..
```links = re.findall('<td class="songs-bitrate-1"><i class="fa fa-music"></i>(.+?)<em>',content.text)```
change content to content.text in re.findall() second parameter.